README.md

```diff
@@ -102,4 +153,5 @@ Get statistics of task
 
 ## Testing
 
-Simply start `tarantool test.lua`
+Tap tests `tarantool test.lua`  
+Luatests tests `luatest -v`
```

Comment:
We have a Makefile that does everything we need for testing, so it's enough to run command `make test` for testing expirationd.

Reply:
Now both(luatest, taptest) and those tests are launched in the make